packages/loader/container-loader/src/container.ts

```diff
@@ -579,6 +579,16 @@ export class Container
 		return this._lifecycleState === "disposing" || this._lifecycleState === "disposed";
 	}
 
+	/**
+	 * The error that caused the container to close or dispose.
+	 *
+	 * @remarks If the container is closed and then disposed, both with errors given, this will expose the close error only.
```

Comment:
I think we can use `/** {@inheritDoc IContainer.closedWithError} */` here to keep them in sync.

Reply:
Not sure what's up with the markdown link in there, I get red squiggles if I try that.  Add a suggestion if I got it wrong.

Reply:
PS - I wish VS Code understood `@inheritDoc`... maybe it's trying?  It's dropping in half the comment from the interface when I hover over a reference on the class.

Reply:
iherit doc does nothing here, as Container isn't exported, so i wouldn't even bother.